lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -28,6 +27,12 @@
 #include "magma_logging.h"
 #include "Utilities.h"
 #include "DiameterCodes.h"
+#include <sys/socket.h>
```

Comment:
nit: Not related to these changes, but `SessionState.h`  file should be the top include in the implementation file

Reply:
Adopted the change.